protocol/protobuf/application_metadata_message.proto

```diff
@@ -27,7 +27,7 @@ message ApplicationMetadataMessage {
     DECLINE_REQUEST_TRANSACTION = 11;
     SYNC_INSTALLATION_CONTACT_V2 = 12;
     SYNC_INSTALLATION_ACCOUNT = 13;
-    SYNC_INSTALLATION_PUBLIC_CHAT = 14;
+    SYNC_INSTALLATION_CHAT = 14;
```

Comment:
I think this is breaking backwards compatibility.

With this change new instances still might receive `SYNC_INSTALLATION_PUBLIC_CHAT` from an older instance and will try to parse it as `SYNC_INSTALLATION_CHAT`, because it's also `14`.

- `SYNC_INSTALLATION_CHAT` should be defined with a new sequential number.
- `14` can be marked as `reserved`, if we're deprecate it, or leave as is if we want some backwards compatibility. But then we should also keep `message SyncInstallationPublicChat` and the it's handler.

https://protobuf.dev/programming-guides/proto3/#fieldreserved